x/ccv/provider/simulation/genesis.go

```diff
@@ -0,0 +1,44 @@
+package simulation
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/interchain-security/v5/x/ccv/provider/types"
+)
+
+// Simulation parameter constants
+const (
+	// only includes params that make sense even with a single
+	maxProviderConsensusValidators = "max_provider_consensus_validators"
+)
+
+// genMaxProviderConsensusValidators returns randomized maxProviderConsensusValidators
+func genMaxProviderConsensusValidators(r *rand.Rand) int64 {
+	return int64(r.Intn(250) + 1)
+}
+
+// RandomizedGenState generates a random GenesisState for staking
+func RandomizedGenState(simState *module.SimulationState) {
```

Comment:
This is not a test, this is the genesis setup for the simulator

Reply:
it is supposed to initialize parameters at random. see this in the gov module: https://github.com/cosmos/cosmos-sdk/blob/a32186608aab0bd436049377ddb34f90006fcbf7/x/gov/simulation/genesis.go#L28

Reply:
Can we expand it for more params besides `MaxProviderConsensusValidators`. It can be part of another PR.

Reply:
Yeah we can, it's a question of what benefit it gives us. We aren't checking anything about the other parameters, and in fact most are completely unused without consumer chains. I want to avoid putting lots of stuff here that just does nothing in relaity